src/hooks/useCurrentUserTenantsPermissions/useCurrentUserTenantsPermissions.js

```diff
@@ -39,7 +43,7 @@ export const useCurrentUserTenantsPermissions = (params = {}, options = {}) => {
       }, {});
     },
     {
-      enabled: Boolean(tenants?.length),
+      enabled: Boolean(tenants?.length) && !!stripes.user?.user?.consortium,
```

Comment:
`Boolean(tenants?.length && stripes.user?.user?.consortium)`

but curious, how is it possible to use consortium manager not in consortium?

Reply:
We are facing an issue where user.consortium is not being settled into the stripes at the moment hook is called. As a result, the POST request to the publications API uses the URL ${CONSORTIA_API}/${consortium?.id}/${PUBLICATIONS_API}, but consortium.id is undefined